src/test/ui/lint/unaligned_references_external_macro.stderr

```diff
@@ -0,0 +1,46 @@
+error: reference to packed field is unaligned
+  --> $DIR/unaligned_references_external_macro.rs:5:1
+   |
+LL | / unaligned_references_external_crate::mac! {
+LL | |
+LL | |
+LL | |     #[repr(packed)]
+...  |
+LL | |     }
+LL | | }
+   | |_^
+   |
+note: the lint level is defined here
+  --> $DIR/unaligned_references_external_macro.rs:5:1
+   |
+LL | / unaligned_references_external_crate::mac! {
+LL | |
+LL | |
+LL | |     #[repr(packed)]
+...  |
+LL | |     }
+LL | | }
+   | |_^
+   = note: fields of packed structs are not properly aligned, and creating a misaligned reference is undefined behavior (even if that reference is never dereferenced)
+   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
+
+warning: borrow of packed field is unsafe and requires unsafe function or block (error E0133)
```

Comment:
Can you `allow` this lint, just to make sure we focus on the one we really care about here?

Reply:
That makes sense. Updated to allow `safe_packed_borrows`.